compiler/util/llvmGlobalToWide.cpp

```diff
@@ -1663,7 +1697,8 @@ static
 bool containsGlobalPointers(unsigned gSpace, SmallSet<Type*, 10> & set, Type* t)
 {
   // All primitive types do not need to change.
-  if(t->isPrimitiveType() || t->isIntegerTy()) return false;
+  if(t->isFloatingPointTy() || t->isX86_MMXTy() || t->isLabelTy() ||
+     t->isMetadataTy() || t->isVoidTy() || t->isIntegerTy()) return false;
   assert(!t->isVoidTy());
```

Comment:
Since `t->isVoidTy()` check is in the if stmt above, I think this assert() is no longer necessary.